test/util/testnode/config.go

```diff
@@ -137,8 +137,8 @@ func DefaultParams() *tmproto.ConsensusParams {
 
 func DefaultTendermintConfig() *tmconfig.Config {
 	tmCfg := tmconfig.DefaultConfig()
-	// Reduce the target height duration so that blocks are produced faster
-	// during tests.
+	// TimeoutCommit sets the maximum duration the application waits for transactions to be committed. 
```

Comment:
```suggestion
	// TimeoutCommit is the duration the node waits after committing a block before starting the next height
```
I think this is more accurate